Vladimir Kulikov/sass/flexbox.scss

```diff
@@ -0,0 +1,225 @@
+@import 'mixins';
+@import 'variables';
+@import 'placeholders';
+
+@include font('ProximaNova-Regular');
+@include font('ProximaNova-Light');
+@include font('SourceSansPro-Light');
+@include font('SourceSansPro-SemiBold');
+@include font('OpenSans-Regular');
+
+html {
+    font-size: #{$root-font-size}px;
+}
+
+body {
+    margin: 0;
+}
+
+.container {
+    @include flex(center, center);
+    height: 100vh;
+    background: $body-bg-color;
+}
+
+.weather-widget {
+    width: 389px;
+    border-radius: $radius;
+    box-shadow: 0 21px 19px $shadow-color;
+
+    &__top {
+        background: $top-bg-color;
+        border-radius: $radius $radius 0 0;
+    }
+
+    &__bottom {
+        color: #fff;
+        background: $bottom-bg-color;
+        border-radius: 0 0 $radius $radius;
+    }
+
+    &__image {
+        position: relative;
+        padding: 17.5% 0 13.5% 0;
+        @include flex(center, center);
+    
+        &-refresh {
+            position: absolute;
+            top: 7%;
+            right: 5%;
+            cursor: pointer;
+            background: none;
+            padding: 0;
+            outline: none;
+            border: none;
+        }
+    }
+
+    &__top-info {
+        @include flex(space-between, center);
+        padding-bottom: 5%;
+        font-family: 'ProximaNova-Light', 'Arial', sans-serif;
+        color: #8d9396;
+    }
+
+    &__degrees {
+        @include flex;
+        width: 25%;
+        padding-left: 20px;
+        @include rem(57.6);
+
+        &-symbol {
+            display: block;
+            padding-top: 7%;
+            padding-left: 7%;
+            @include rem(22);
+        }
+    }
+
+    &__text {
+        @include flex;
+        flex-direction: column;
+        width: 53.5%;
+    }
+
+    &__sky {
+        font-family: 'SourceSansPro-Light', 'Arial', sans-serif;
+        @include rem(24);
+    }
+
+    &__location {
+        font-family: 'SourceSansPro-SemiBold', 'Arial', sans-serif;
+        @include rem(16);
+    }
+
+    &__date {
+        @include flex(center, center);
+        flex-direction: column;
+        width: 16.5%;
+        padding: 13px 0;
+        font-family: 'OpenSans-Regular', 'Arial', sans-serif;
+        font-weight: bold;
+        text-transform: uppercase;
+        text-align: center;
+        color: #fff;
+        background: $date-bg-color;
+
+        &-day {
+            @include rem(18);
+        }
+    }
+    
+    &__bottom-info {
+        @include flex(space-between);
+        padding: 10% 10%;
+    }
+
+    &__wind {
+        @extend %bottom-icon;
+        &::before {
+            @include icon(28px, 18px, 88px);
+        }
+    }
+
+    &__humidity {
+        @extend %bottom-icon;
+        &::before {
+            @include icon(16px, 20px, 49px);
+        }
+    }
+
+    &__clear {
+        @extend %bottom-icon;
```

Comment:
You can do like this, it's up to you.
In my opinion it's easier to add the same class for these elements and describe this class here